src/components/Homepage/Topics.js

```diff
@@ -0,0 +1,27 @@
+function Topics() {
```

Comment:
Topics is a little ambiguous for the component name, maybe change it to feed?

Reply:
Okay, I will re-name it to feed.